### DISCUSSION TITLE
OpenAI Assistants

### DISCUSSION THREAD

**Discussion by jduprat6** [FlowiseAI/Flowise#1209] [Q&A]
I've created an open AI assistant in Flowise however the uploaded .txt files are yellow and when I ask the bot about the uploaded files the bot says "myfiles_browser tool is not accessible"  I have this deployed on render.  This seems to work when testing the chat directly in Flowise but it does not work when I use the "Share Chatbot" link

**Comment by HenryHengZJ** [comment]
can you try redeploy Render with latest commit and verify if the issues still persists? just merged in a bugfix - https://github.com/FlowiseAI/Flowise/pull/1207

**Reply by jduprat6** [reply]
Thank you. What I did was upload the files directly in OpenAI and then loaded the existing Assistant from OpenAI and that worked. I will also redeploy

**Comment by SardarAwais88** [comment]
how can i load existing assistant in flowise its client project bu i dont know how to load the existing assistant
